### DISCUSSION TITLE
Replace deprecated material design brand icons with simple icons

### DISCUSSION THREAD

**Discussion by metbril** [home-assistant/frontend#15876] [Other feature requests]
The other day I noticed a big red deprecation notice with the [Material Design Icons library](https://pictogrammers.com/library/mdi/). All brand icons are currently deprecated and will be removed in v8.

https://github.com/Templarian/MaterialDesign/issues/6602

This includes the Home Assistant logo itself.

Do you already have any plans to add official support for a library of brand icons, for example [simpleicons](https://simpleicons.org/)?

**Comment by ravngr** [comment]
There was a [forum thread](https://community.home-assistant.io/t/material-design-icons-simpleicons-add-support-for-simpleicons-library/108765) a few years back that got admin attention, though it doesn't look like an issue was opened as a result (not that I could find) and discussion got a bit sidetracked. There was also a follow-up [feature request](https://community.home-assistant.io/t/support-simpleicons/111554) and a couple of [custom](https://github.com/vigonotion/hass-simpleicons) [integrations](https://github.com/dannyqwertz/hass-simpleicons) to add simpleicons.

It might be good to start identifying problematic icons before the dependency updates. The issues estimates the release of v8 sometime this year.

**Comment by metbril** [comment]
In the [Brand/Logo category](https://pictogrammers.com/library/mdi/category/brand-logo/), currently at 242 icons, I can see for example:

- https://pictogrammers.com/library/mdi/icon/home-assistant/ !!!
- https://pictogrammers.com/library/mdi/icon/z-wave/ !!!
- https://pictogrammers.com/library/mdi/icon/zigbee/ !!!
- https://pictogrammers.com/library/mdi/icon/android/
- https://pictogrammers.com/library/mdi/icon/apple/
- https://pictogrammers.com/library/mdi/icon/apple-ios/
- https://pictogrammers.com/library/mdi/icon/github/
- https://pictogrammers.com/library/mdi/icon/microsoft-visual-studio-code/

And there are probably others used throughout Home Assistant Core (integrations) and elsewhere?

**Reply by ChristophCaina** [reply]
in this case, it could be an option to implement "custom band icons" (https://github.com/elax46/custom-brand-icons) which is available on HACS either into the core or as a fallback option (?)

**Reply by metbril** [reply]
I would prefer the simpleicons library which is widely known and supported. And also referred to by the MDI team.
